docs/java/features/resilience/resilience.md

```diff
@@ -0,0 +1,200 @@
+---
```

Comment:
I use an offline spellchecker & VS Code for writing. I don't see a VS Code plugin for Grammarly. And I am hesitant to install it to my browser since it can read just about any data of all sites I visit.

How do you run it?

Reply:
Well, I run it as a web-app freemium and it's damn good! I'd be happy if SAP paid me that 70 bucks yearly license to use the pro version. Your spelling is Ok, but that's not everything that can be caught by the NLP tool like Grammarly. 
![image](https://user-images.githubusercontent.com/10460752/86289919-cb8e5480-bbec-11ea-97ae-65b1fad96235.png)
Make an account and give it a try. I do use it as a browser plugin, but by the settings, it should read only what I type. I don't what to have any polemics about it, but having some standard is good to avoid needless discussion.